@navikt/core/react/src/progress-bar/ProgressBar.tsx

```diff
@@ -0,0 +1,129 @@
+import cl from "clsx";
+import React, { HTMLAttributes, forwardRef, useEffect, useState } from "react";
+
+interface ProgressBarPropsBase
+  extends Omit<HTMLAttributes<HTMLDivElement>, "role"> {
+  /**
+   * Changes height of progressbar.
+   * @default "medium"
+   */
+  size?: "large" | "medium" | "small";
+  /**
+   * Value of progressbar.
+   */
+  value: number;
+  /**
+   * Minimum progress.
+   * @default 0
+   */
+  valueMin?: number;
+  /**
+   * Maximum progress.
+   */
+  valueMax: number;
+  /**
+   * Used to approximate a task duration in seconds.
+   * ProgressBar shows an indeterminate animation after duration is done.
+   */
+  duration?: number;
+  /**
+   * String ID of the element that labels the progress-bar.
+   * Not needed if `aria-label` is used.
+   */
+  "aria-labelledby"?: string;
+  /**
+   * String value that labels the progress-bar.
+   * Not needed if `aria-labelledby` is used.
+   */
+  "aria-label"?: string;
+}
+
+export type ProgressBarProps = ProgressBarPropsBase &
+  (
+    | {
+        "aria-labelledby": string;
+        "aria-label"?: never;
+      }
+    | {
+        "aria-label": string;
+        "aria-labelledby"?: never;
+      }
+  );
+
+/**
+ * ProgressBar
+ * @see [📝 Documentation](https://aksel.nav.no/komponenter/core/progress-bar)
+ * @see 🏷️ {@link ProgressBarProps}
+ *
+ * @example
+ * // For loading content
+ * <ProgressBar value={20} valueMin={0} valueMax={100} duration={12} />
+ *
+ * @example
+ * // As a progress indicator for applications and similar
+ * <ProgressBar value={2} valueMin={0} valueMax={7} />
```

Comment:
Hva menes med "for applications and similar"?

Reply:
Hm, søknader (applications) og f.eks. spørreundersøkelser